test/cpp/test_symint.cpp

```diff
@@ -169,10 +169,8 @@ TEST(SymintTest, TestDynamicSymintArithmetic) {
       std::vector<torch::lazy::Shape>{
           torch::lazy::Shape(torch::kFloat, {10, 20, 30})};
 
-  torch::lazy::NodePtr abs_node =
-      torch::lazy::MakeNode<Abs>(expand_value, std::move(abs_lazy_shapes));
-  torch::lazy::NodePtr relu_node =
-      torch::lazy::MakeNode<Relu>(expand_value, std::move(relu_lazy_shapes));
+  torch::lazy::NodePtr abs_node = torch::lazy::MakeNode<Abs>(expand_value);
```

Comment:
Should we remove `abs_lazy_shapes` and `relu_lazy_shapes` as well?

Reply:
Updated